cibuild.cmd

```diff
@@ -55,6 +53,7 @@ git show --no-patch --pretty=raw HEAD
 
 REM Restore the NuGet packages
 call "%RoslynRoot%\Restore.cmd" || goto :BuildFailed
+call "%RoslynRoot%SetDevCommandPrompt.cmd" || goto :BuildFailed
```

Comment:
Ah, I missed the `setlocal` call in `Restore.cmd`, that would  be why we have to call this a second time here.